File: python/getEmail.py
#rangos de facturacion de una cuenta

import api

# Datos basicos
USR = "elusuario"
PWD = "lacontrasena"
URL = "https://playground.lafactura.co/api2.0"

# Solicitud
solicitud= {
              'getEmail':{
                'id':'900900900'
              }
            }

# Envio
response  = api.getEmail(URL,USR,PWD,solicitud)
# Resultado
print(response.text)

File: php/getEmail.php
<?php
  //Este ejemplo permite conocer el correo electronico habilitado para facturacion electronica ante la DIAN

  include_once("API/api.php");

  $usuario="elusuario"; //coloca en este punto tu usuario
  $contrasena="lacontrasena"; //coloca tu contraseña
  $url="https://playground.lafactura.co/api2.0"; //estea esl la URL de acceso a la api

  //solicita envio de correo
  $solicitud=[
    "getEmail"=>[
        "id"=>"900900900" //documento NIT / CC del que deseamos conocer el correo
    ]
  ];
  //convierto a JSON la solicitud
  $solicitud=json_encode($solicitud,JSON_UNESCAPED_UNICODE | JSON_HEX_QUOT);

  $api=new LF_API_v200($url,$usuario,$contrasena);
  $api->send(LF_SERVICE_GENERAL,$solicitud,LF_METHOD_GET); //metodo GET
  echo $api->response; //resulado de la peticion en JSON

?>
